c2cgeoportal/scaffolds/create/vars_+package+.yaml

```diff
@@ -0,0 +1,127 @@
+extends: CONST_vars.yaml
+
+vars:
+    # Database name
+    db: to be defined
+
+    # The visible host name
+    host: to be define
+
+    # The application's default language. This is the language used by
+    # the application if no specific language is specified in the URLs.
+    # This also defines the language used for the text search.
+    default_locale_name: fr
+
+    # The set of languages supported by the applications.
+    available_locale_names:
+    - en
+    - fr
+    - de
+
+    admin_interface:
+        # Default values for the admin interface's maps.
+        map_x: 740000
+        map_y: 5860000
+        map_zoom: 10
+
+        # The list of functionalities that can be configured
+        # through the admin interface.
+        available_functionalities:
+        - default_basemap
+        - print_template
+        - mapserver_substitution
+
+    functionalities:
+        # Functionalities that are accessible to anonymous
+        # users.
+        anonymous:
+            print_template:
+            - 1 A4 portrait demo
+            - 2 A3 portrait demo
+            default_basemap: plan
```

Comment:
Well, ortho is a special baselayer. Can we really use it as the default basemap?
I meant something in english, not in french like "plan".